Gemfile

```diff
@@ -12,7 +12,7 @@ gem 'retriable', '~> 2.0'
 
 # Workflow and build tools
 gem 'rake', '~> 10.3'
-gem 'bundler', '~> 1.6'
+gem 'bundler', '~> 2.0'
```

Comment:
This needs to be bundler "1.17.3".  Bundler 2.0 is not allowed on Test Cloud.

Reply:
Done